resources/google-music-electron-small.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Renaming: that's fine.

Small would be for the main application icon at the smallest size (16px normal, 32px retina), so the logo retains is visibility and distinctiveness at that size.
I try to follow some of the concepts in this: https://developer.apple.com/library/mac/documentation/UserExperience/Conceptual/OSXHIGuidelines/Designing.html